DrawerKit/DrawerKit/Internal API/PresentationController.swift

```diff
@@ -5,18 +5,25 @@ final class PresentationController: UIPresentationController {
     let inDebugMode: Bool
     let handleView: UIView?
 
+    let presentingDrawerAnimationActions: DrawerAnimationActions
+    let presentedDrawerAnimationActions: DrawerAnimationActions
+
     var drawerFullExpansionTapGR: UITapGestureRecognizer?
     var drawerDismissalTapGR: UITapGestureRecognizer?
     var drawerDragGR: UIPanGestureRecognizer?
     var lastDrawerState: DrawerState = .collapsed
 
     init(presentingVC: UIViewController?,
+         presentingDrawerAnimationActions: DrawerAnimationActions,
          presentedVC: UIViewController,
+         presentedDrawerAnimationActions: DrawerAnimationActions,
```

Comment:
Ditto.

Reply:
@RuiAAPeres Similar argument as that presented in https://github.com/Babylonpartners/DrawerKit/pull/36#discussion_r149625323.